internal/index/indexscanner/scanner_test.go

```diff
@@ -90,6 +94,58 @@ func TestReadPluginFile(t *testing.T) {
 	}
 }
 
+func TestReadReceiptFromFile(t *testing.T) {
+	type args struct {
+		receiptFilePath string
+	}
+	tests := []struct {
+		name       string
+		args       args
+		wantStatus index.ReceiptStatus
+	}{
+		{
+			name: "read receipt of plugin without status",
+			args: args{
+				receiptFilePath: filepath.Join(testdataPath(t), "testindex", "receipts", "foo.yaml"),
+			},
```

Comment:
You could also use your testdata builder `NewReceipt().WithXX().V()` and store that in a temporary directory with `tmpDir.WriteYAML(...)`.
Why did you opt for test fixtures instead?

Reply:
Ya I can change it to this. I think I did it this way because this is how it was outlined in the previous receipts PR and I just had that on my mind while I was working on this initially.

Reply:
Yeah, we talked about both options. Back then we didn't have the receipt builder, so the static file was easier. But now, I think the dynamic approach is far better.